### PR TITLE
updated DSAdmin page for Docker

### PR DESCRIPTION
**Pull request by jayam04**
Added Instructions to run DSAdmin in Docker Setup.

### CODE REVIEW

File: Debugging-datastore-locally.md
Comment (seanlip) [inline Debugging-datastore-locally.md]:
```suggestion
## How to start DSAdmin
```

**Review by seanlip — APPROVED** [review]
Thanks @jayam04! Just one small suggested fix if that's OK.

**Comment by jayam04** [comment]
# Note!!
Merge after [PR#20425 in Oppia](https://github.com/oppia/oppia/pull/20425) gets merged.

**Comment by jayam04** [comment]
@seanlip Updated with required changed!!

**Review by seanlip — APPROVED** [review]
Thanks, LGTM! Merge whenever you like :)

**Comment by jayam04** [comment]
Thanks! I will merge once other PR gets merged.

**Comment by seanlip** [comment]
@jayam04 This has been open for 2 weeks, are we planning to merge it?

**Comment by jayam04** [comment]
The PR for changes is not yet merged, but we can merge this if you want as
main PR needs final review.

**Comment by seanlip** [comment]
@jayam04 Ah -- no, I think we should get the other PR merged so that this page is correct and we don't confuse contributors. What's blocking the other PR, can we get that in soon?

**Comment by jayam04** [comment]
@seanlip, this PR can be merged now.

**Comment by jayam04** [comment]
@U8NWXD Can you please commit this PR.

**Pull request merged**
